SQL/0000-00-03-ConfigTables.sql

```diff
@@ -240,4 +241,6 @@ INSERT INTO Config (ConfigID, Value) SELECT ID, "adniT1" FROM ConfigSettings cs
 INSERT INTO Config (ConfigID, Value) SELECT ID, 19 FROM ConfigSettings cs WHERE cs.Name="reject_thresh";
 INSERT INTO Config (ConfigID, Value) SELECT ID, "/opt/niak-0.6.4.1/" FROM ConfigSettings cs WHERE cs.Name="niak_path";
 INSERT INTO Config (ConfigID, Value) SELECT ID, "INTERLACE_outputDWIFileNameSuffix" FROM ConfigSettings cs WHERE cs.Name="QCed2_step";
+INSERT INTO Config (ConfigID, Value) SELECT ID, "localizer" FROM ConfigSettings cs WHERE cs.Name="excluded_series_description";
+INSERT INTO Config (ConfigID, Value) SELECT ID, "scout" FROM ConfigSettings cs WHERE cs.Name="excluded_series_description";
```

Comment:
@cmadjar 
do we want to change the current default behavior for existing projects, whereby now we exclude 'localizer' and with this patch, we exclude both 'localizer' and 'scout'?

Reply:
@MounaSafiHarab technically, if the idea is to exclude the localizers, scouts should also be excluded. If it were the case by default in the past, we would have had a lot less violated scans in PREVENT-AD ;)
Either works for me. If you prefer that I remove the scout entry from the default behaviour, I can do it.